static/js/task/controllers/task.controller.js

```diff
@@ -58,12 +56,41 @@
                         ).finally(function () {
                         });
                     }
+                    countdown("countdown1",0,self.time_left);
                 },
                 function error(data) {
                     $mdToast.showSimple('Could not get task with data.');
                 });
         }
 
+        function countdown(elementName, minutes, seconds )
```

Comment:
Can you convert this into angular directive instead? Avoid manipulating DOM directly from controller.